lib/charms/observability_libs/v1/cert_handler.py

```diff
@@ -283,6 +283,7 @@ def __init__(
         peer_relation_name: str = "peers",
         cert_subject: Optional[str] = None,
         sans: Optional[List[str]] = None,
+        refresh_events: List[BoundEvent] = [],
```

Comment:
We shouldn't use mutable default argument.

```suggestion
        refresh_events: Optional[List[BoundEvent]] = None,
```

Reply:
yup, I was suggesting using `()`

Reply:
yeah, but `pyright` would complain. I'll keep it as `None` and add a check if `not None`